package.json

```diff
@@ -1,73 +1,86 @@
-{
-  "name": "bikely",
-  "version": "0.0.1",
-  "description": "",
-  "author": "",
-  "repository": "https://github.com/HubertLegec/bikely.git",
-  "private": false,
-  "license": "UNLICENSED",
-  "scripts": {
-    "prebuild": "rimraf dist",
-    "build": "nest build",
-    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
-    "start": "nest start",
-    "start:dev": "nest start --watch",
-    "start:debug": "nest start --debug --watch",
-    "start:prod": "node dist/main",
-    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
-    "test": "jest",
-    "test:watch": "jest --watch",
-    "test:cov": "jest --coverage",
-    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
-  },
-  "dependencies": {
-    "@nestjs/common": "^7.5.1",
-    "@nestjs/core": "^7.5.1",
-    "@nestjs/platform-express": "^7.5.1",
-    "@nestjs/swagger": "^4.7.13",
-    "reflect-metadata": "^0.1.13",
-    "rimraf": "^3.0.2",
-    "rxjs": "^6.6.3",
-    "swagger-ui-express": "^4.1.6"
-  },
-  "devDependencies": {
-    "@nestjs/cli": "^7.5.1",
-    "@nestjs/schematics": "^7.1.3",
-    "@nestjs/testing": "^7.5.1",
-    "@types/express": "^4.17.8",
-    "@types/jest": "^26.0.15",
-    "@types/node": "^14.14.6",
-    "@types/supertest": "^2.0.10",
-    "@typescript-eslint/eslint-plugin": "^4.6.1",
-    "@typescript-eslint/parser": "^4.6.1",
-    "eslint": "^7.12.1",
-    "eslint-config-prettier": "7.2.0",
-    "eslint-plugin-prettier": "^3.1.4",
-    "jest": "^26.6.3",
-    "prettier": "^2.1.2",
-    "supertest": "^6.0.0",
-    "ts-jest": "^26.4.3",
-    "ts-loader": "^8.0.8",
-    "ts-node": "^9.0.0",
-    "tsconfig-paths": "^3.9.0",
-    "typescript": "^4.0.5"
-  },
-  "jest": {
-    "moduleFileExtensions": [
-      "js",
-      "json",
-      "ts"
-    ],
-    "rootDir": "src",
-    "testRegex": ".*\\.spec\\.ts$",
-    "transform": {
-      "^.+\\.(t|j)s$": "ts-jest"
-    },
-    "collectCoverageFrom": [
-      "**/*.(t|j)s"
-    ],
-    "coverageDirectory": "../coverage",
-    "testEnvironment": "node"
-  }
-}
+{

```

Comment:
takie zmiany wcięć to słaby pomysł, nie wiadomo co tak na prawdę ważnego się zmieniło w pliku